vms/avm/environment_test.go

```diff
@@ -69,6 +76,13 @@ var (
 
 	keys  = secp256k1.TestKeys()[:3] // TODO: Remove [:3]
 	addrs []ids.ShortID              // addrs[i] corresponds to keys[i]
+
+	noFeesTestConfig = &config.Config{
```

Comment:
some unit tests currently requires zero fees since they "manually" build transactions. I have an incoming PR that consolidates the way we build txs and uses proper tx builders. When we'll have that, we'll remove the `noFeeTestConfig`.